src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/ContainerProcessor.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.blobrouter.tasks.processors;
+
+import com.azure.storage.blob.BlobContainerAsyncClient;
+import com.azure.storage.blob.models.BlobItem;
+import org.slf4j.Logger;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+public class ContainerProcessor {
+
+    private static final Logger LOGGER = getLogger(ContainerProcessor.class);
+
+    public void process(BlobContainerAsyncClient containerClient) {
+        String containerName = containerClient.getBlobContainerName();
+        AtomicInteger processedBlobCount = new AtomicInteger(0);
```

Comment:
how this will work if there will be  multiple instance?

Reply:
still fine in the context of single instance.

Reply:
it may print same number many times?

Reply:
each instance will process different files and log how many files this particular instance has processed

Reply:
when pod crashed new pod starts from zero, what is the point?

Reply:
This counter is for a single job run. The purpose is to know how many files were processed within that job run.